lambdas/jwtverification/src/main/java/uk/gov/di/ipv/cri/passport/jwtverification/JwtVerificationHandler.java

```diff
@@ -13,9 +15,14 @@
 import com.nimbusds.oauth2.sdk.util.StringUtils;
 import uk.gov.di.ipv.cri.passport.library.service.ConfigurationService;
 
+import java.io.ByteArrayInputStream;
```

Comment:
Could you remove some of these unused imports here

Reply:
Optimised